### PR TITLE
Update `dynamodb.TableEventSubscriptionArgs` with latest Event Source Mapping params

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #950

### CODE REVIEW

File: sdk/nodejs/dynamodb/dynamodbMixins.ts [hunk 1]
```diff
@@ -28,10 +28,38 @@ export interface TableEventSubscriptionArgs {
      */
     readonly batchSize?: number;
```
Comment (mikhailshilkov) [inline sdk/nodejs/dynamodb/dynamodbMixins.ts:29]: Do we want to add these?

```
readonly destinationConfig?: pulumi.Input<inputs.lambda.EventSourceMappingDestinationConfig>;
readonly startingPositionTimestamp?: pulumi.Input<string>;
```

File: sdk/nodejs/dynamodb/dynamodbMixins.ts [hunk 2]
```diff
@@ -28,10 +28,38 @@ export interface TableEventSubscriptionArgs {
...
+
+    /**
+     * The maximum age of a record that Lambda sends to a function for processing. Minimum of `60`, maximum and default of `604800`
+     */
+    readonly maximumRecordAgeInSeconds?: number;
+
+    /**
+     * If the function returns an error, split the batch in two and retry. Defaults to `false`.
+     */
+    readonly bisectBatchOnFunctionError?: boolean;
```
Comment (mikhailshilkov) [inline sdk/nodejs/dynamodb/dynamodbMixins.ts:61]: Do you mind sorting them alphabetically to be able to easily compare to the source list? Thanks!

Comment (stack72) [reply sdk/nodejs/dynamodb/dynamodbMixins.ts:61]: done!

File: sdk/nodejs/dynamodb/dynamodbMixins.ts (hunk 1 above)
Comment (stack72) [reply sdk/nodejs/dynamodb/dynamodbMixins.ts:29]: starting position timestamp is only for kinesis

Comment (mikhailshilkov) [reply sdk/nodejs/dynamodb/dynamodbMixins.ts:29]: Ah, okay. And `destinationConfig`?

Comment (stack72) [reply sdk/nodejs/dynamodb/dynamodbMixins.ts:29]: I DMed you in Slack :)

File: sdk/nodejs/dynamodb/dynamodbMixins.ts [hunk 3]
```diff
@@ -101,6 +134,12 @@ export class TableEventSubscription extends lambda.EventSubscription {
             eventSourceArn: streamArn,
             functionName: this.func.name,
             startingPosition: args.startingPosition,
+            maximumBatchingWindowInSeconds: args.maximumBatchingWindowInSeconds,
+            parallelizationFactor: args.parallelizationFactor,
+            maximumRetryAttempts: args.maximumRetryAttempts,
+            maximumRecordAgeInSeconds: args.maximumRecordAgeInSeconds,
+            bisectBatchOnFunctionError: args.bisectBatchOnFunctionError,
+            destinationConfig: args.destinationConfig,
```
Comment (mikhailshilkov) [inline sdk/nodejs/dynamodb/dynamodbMixins.ts:142]: Nit: sort these too

File: sdk/nodejs/kinesis/kinesisMixins.ts [hunk 4]
```diff
@@ -26,6 +27,38 @@ export interface StreamEventSubscriptionArgs {
...
+    /**
+     * The maximum number of times to retry when the function returns an error. Minimum of `0`, maximum and default of `10000`.
+     */
+    readonly maximumRetryAttempts?: number;
+
+    /**
+     * The number of batches to process from each shard concurrently. Minimum and default of `1`, maximum of `10`
+     */
+    readonly parallelizationFactor?: number;
+
```
Comment (mikhailshilkov) [inline sdk/nodejs/kinesis/kinesisMixins.ts:61]: So do we want `startingPositionTimestamp` in Kinesis?

File: sdk/nodejs/kinesis/kinesisMixins.ts [hunk 5]
```diff
@@ -97,6 +130,12 @@ export class StreamEventSubscription extends lambda.EventSubscription {
             functionName: this.func.name,
             startingPosition: args.startingPosition,
             startingPositionTimestamp: args.startingPositionTimestamp,
+            maximumBatchingWindowInSeconds: args.maximumBatchingWindowInSeconds,
+            parallelizationFactor: args.parallelizationFactor,
+            maximumRetryAttempts: args.maximumRetryAttempts,
+            maximumRecordAgeInSeconds: args.maximumRecordAgeInSeconds,
+            bisectBatchOnFunctionError: args.bisectBatchOnFunctionError,
+            destinationConfig: args.destinationConfig,
```
Comment (mikhailshilkov) [inline sdk/nodejs/kinesis/kinesisMixins.ts:138]: Nit: sort

File: sdk/nodejs/kinesis/kinesisMixins.ts (hunk 4 above)
Comment (stack72) [reply sdk/nodejs/kinesis/kinesisMixins.ts:61]: It's already in there ;)

File: sdk/nodejs/dynamodb/dynamodbMixins.ts (hunk 3 above)
Comment (stack72) [reply sdk/nodejs/dynamodb/dynamodbMixins.ts]: Fixed!

File: sdk/nodejs/kinesis/kinesisMixins.ts (hunk 5 above)
Comment (stack72) [reply sdk/nodejs/kinesis/kinesisMixins.ts]: Fixed!

File: sdk/nodejs/kinesis/kinesisMixins.ts (hunk 4 above)
Comment (mikhailshilkov) [reply sdk/nodejs/kinesis/kinesisMixins.ts:61]: Oh, f*ck, sorry

**Pull request merged**